Makefile

```diff
@@ -38,6 +38,7 @@ BASH=
 endif
 
 CLANG_VERSION=$(shell $(BASH) ./llvm_version.sh $(LLVM_PROJ_DIR))
+CLANG_MAJOR_VERSION=$(shell $(BASH) ./llvm_version_major.sh $(LLVM_PROJ_DIR))
```

Comment:
I don't think we need both of these.. can you just modify the existing llvm_version.sh to report just the major version?'

Otherwise LGTM

Reply:
Take a look at that last commit: the "make sure this isn't version 9 or 10" code also needed to change. (I wonder if there is an even better way to do that?)